src/main/java/frc/robot/controls/Operator.java

```diff
@@ -1,15 +1,48 @@
 package frc.robot.controls;
 
+import frc.robot.commands.MoveToHeight;
+import frc.robot.commands.ResetEncoderAtBottom;
 import frc.robot.commands.arm.ExtendToPosition;
 import frc.robot.constants.ArmConstants;
+import frc.robot.constants.ElevatorConstants;
 import frc.robot.constants.OIConstants;
+import frc.robot.subsystems.Elevator;
 import frc.robot.subsystems.FourBarArm;
 import lib.controllers.GameController;
 import lib.controllers.GameController.Button;
+import lib.controllers.GameController.DPad;
+import lib.controllers.GameController.Axis;;
 
 public class Operator {
   private static GameController operator = new GameController(OIConstants.kOperatorJoy);
 
+  public static void configureControls(Elevator elevator) {
```

Comment:
Combine this function with the other configureControls() function

Reply:
Code change added.

Reply:
It's still there...
maybe you didn't push?
You can check the code when you reply to my comment

Reply:
sorry, I didn't push. Would you like me to push after I reply to each feedback?